### DISCUSSION TITLE
Fixer for no-mixed-spaces-and-tabs?

### DISCUSSION THREAD

**Discussion by moniuch** [eslint/eslint#15505] [Ideas]
Unless I am missing something, I couldn't find a way to fix `no-mixed-spaces-and-tabs` violations. There is an old ticket (dated 2016) https://github.com/eslint/eslint/issues/6873 where the team couldn't come to a consensus. Not sure what were the options that were in the battlefield, but I have been struggling with this rules and found no answer.

**Comment by nzakas** [comment]
If a rule is fixable, it will say so on the documentation page. Here's the docs for this rule:
https://eslint.org/docs/rules/no-mixed-spaces-and-tabs

We don't plan on adding a fixer for this rule.
